tests/src/main/scala/sttp/tapir/tests/data/Problem.scala

```diff
@@ -0,0 +1,14 @@
+package sttp.tapir.tests.data
+
+import sttp.tapir.{Schema, SchemaType}
+
+case class Problem()
+
+object Problem {
```

Comment:
I think that this class won't be reused in other tests maybe we can put it into companion object in `VerifyYamlTest`

Reply:
moved the class